src/lib/plugins/validation/rule.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, ContentChild, Input, OnChanges, SimpleChanges, TemplateRef } from '@angular/core';
+import { PluginService } from '../plugin.service';
+import { TemplateHostService } from '../../template/template-host.service';
+
+const ruleBindings = [ 'required', 'not_empty_list', 'any_object', 'eq', 'string', 'length_between', 'length_equal', 'min_length',
```

Comment:
we have only for and key are not rules? maybe easier will be to add them as excpetions in ngChanges rather than support this array?